src/brogue/Monsters.c

```diff
@@ -954,7 +954,7 @@ boolean summonMinions(creature *summoner) {
 
     if (canSeeMonster(summoner)) {
         monsterName(monstName, summoner, true);
-        if (monsterText[summoner->info.monsterID].summonMessage) {
+        if (monsterText[summoner->info.monsterID].summonMessage[0]) {
```

Comment:
I suppose your comment about uninitialised structs was concerning this line, and whether or not [0] will be '\0' if not manually specified. According to [cppreference](https://en.cppreference.com/w/c/language/struct_initialization):

>All members that are not initialized explicitly are initialized implicitly the same way as objects that have static storage duration.

And static-stored data is zeroed. So it seems valid to me

Reply:
Yes

Reply:
Is anything else blocking here, or could be merged?